rskj-core/src/main/java/org/ethereum/rpc/HttpUtils.java

```diff
@@ -0,0 +1,18 @@
+package org.ethereum.rpc;
+
+public class HttpUtils {
+
+    public static String getMimeType(String contentTypeValue) {
```

Comment:
Also add a comment explaining that this mimics Netty's implementation, and that we're doing this because we have an older version without this feature.